---
urlcolor: blue
linkcolor: blue
output:
  word_document:
    fig_width: 7
    fig_height: 5
    fig_caption: true
    highlight: "tango"
    reference_docx: word_style.docx
    df_print: "kable"
  html_document:
    fig_width: 7
    fig_height: 5
    fig_caption: true
    highlight: "tango"
---








---
title: "Phylodynamic Analysis: Liège,: 2020-03-31 "
date: April 21, 2020
---





### Primary author: Olivia Boyd

#### Olivia Boyd, Lily Geidelberg, David Jorgensen, Manon Ragonnet, Igor Siveroni, Erik Volz and the [Imperial College COVID-19 Response Team](http://sarscov2phylodynamics.org/about/)

### Report updated on 2020-04-21





## Background information  




#### This is analysis is based on : 
  
* **50 whole genomes** sampled from **within Liège**
* **73 whole genomes** sampled from outside of **Liège**
* Samples within Liège were collected between **2020-03-05** and **2020-03-31**


These numbers will differ from the number of uploaded sequences as we remove sequences with likely sequencing errors or significant gaps. Sequences were deduplicated prior to analyses.

<!-- ##### To add: [optional plot of sample distribution through time] -->



## How many are infected in Liège?

Using a phylodynamic model we estimate epidemiological parameters using SARS-CoV 2 sequence data from Liège together with a background set of sequences sampled from the larger internationational viral population. [The model is explained in detail here](http://whoinfectedwhom.org/seijr0.1.0_methods.pdf). Reported case numbers for comparison are extracted from [Sciensano, the Belgian instute for health](https://epistat.wiv-isp.be/covid/).



![plot of chunk Cumulative estimated infections through time]({{ site.url }}/assets/20200421-114817-ece7e7f5/SEIJR_plot_size.png)


![plot of chunk Cumulative estimated infections through time log scale]({{ site.url }}/assets/20200421-114817-ece7e7f5/SEIJR_plot_size_log.png)


*Figure 1: Estimated cumulative infections through time represented by solid black line (median) and 95% CrI (ribbon). Black points represent reported cases in Liège. The dashed line indicates the date of last sample in Liège in this analysis.*


* Estimated cumulative infections at last sample (2020-03-31): **51120 [10340-207545]** median [95%CI]

* Cumulative confirmed infections reported at 2020-03-31: **1854**  

<!-- * Cumulative number of active infections at 2020-03-31:   -->



![plot of chunk daily estimated infections through time]({{ site.url }}/assets/20200421-114817-ece7e7f5/Daily.png)


![plot of chunk daily estimated infections through time log scale]({{ site.url }}/assets/20200421-114817-ece7e7f5/Daily_log.png)


*Figure 2: Estimated daily  infections through time represented by solid black line (median) and 95% CrI (ribbon). Black points represent reported cases in Liège. The dashed line indicates the date of last sample in Liège in this analysis.*


Towards the end of the curve the estimated number of daily infections starts to decrease. This may be due to the shutting of all non-essential activities on [2020-03-18 across Belgium](https://www.Liège.be/fr/actualites/comment-se-proteger-contre-les-virus-comme-le-coronavirus-covid-19-ou-la-grippe-saisonniere). However, further anlayses are required to assess the impact of non-pharmaceutical interventions on transmission in this region.

![plot of chunk reporting]({{ site.url }}/assets/20200421-114817-ece7e7f5/reporting.png)

*Figure 3: Estimated percentage of daily cases reported in Liège. Error bars represent the 95% credible interval.*


The percentage of estimated cases which were confirmed reported cases between 2020-03-01 and 2020-03-31 ranged between 1.08 to 5.93% with a mean of 4.44%. The mean estimated reporting rate is similar to previous findings [reported by Russell et al using case fatality ratio estimates.](https://cmmid.github.io/topics/covid19/severity/global_cfr_estimates.html) 


![plot of chunk Rt]({{ site.url }}/assets/20200421-114817-ece7e7f5/Rt.png)

*Figure 4: Reproduction number through time. The black vertical dashed line indicates the date of last sample in Liège in this analysis. Orange and red dashed lines indicate dates of school closure and general lockdown in Liège, respectively. *

Reproduction number remains relatively stable prior to the introduction of non-pharmaceutical interventions. Following [school closures (orange dashed line)](https://www.dhnet.be/actu/belgique/fermeture-des-ecoles-a-cause-du-coronavirus-la-ministre-desir-donne-des-precisions-5e6baf919978e201d8b55769) on 2020/03/16 and [general lockdown (red dashed line)](https://www.Liège.be/fr/actualites/comment-se-proteger-contre-les-virus-comme-le-coronavirus-covid-19-ou-la-grippe-saisonniere) being implemented on 2020/03/18, the reproduction number begins to decrease. However, further analyses are still required to assess the strength of effect these interventions may have had on the reproduction number. 

Reproduction number at last sample (2020-03-31): **1.02 [0.214-2.7]** median [95% CrI]


## How quickly has the epidemic in Liège grown?




```


| Quantile | Reproduction number | Growth rate (per day) | Doubling time (days) |
|:--------:|:-------------------:|:---------------------:|:--------------------:|
|   50%    |        3.48         |         0.220         |         3.16         |
|   2.5%   |        3.02         |         0.187         |         2.8          |
|  97.5%   |         3.9         |         0.247         |         3.71         |

Table: Reproduction number, growth rate and doubling times
```






## How has SARS-CoV 2 evolved in Liège?


![plot of chunk mcc_tree]({{ site.url }}/assets/20200421-114817-ece7e7f5/mcc.png)

*Figure 5: Time scaled phylogeny co-estimated with epidemiological parameters. The colour of the tips corresponds to location sampling; red tips were sampled from within Liège, blue tips from outside.*




##### Molecular clock rate of evolution: **0.000957 [0.000698-0.00114]** median [95% CrI]  

<!-- #### (optional) Number of introductions into Liège (someone needs to write code to compute this) -->




## Predicted cumulative infections over next 14 days (assuming exponential growth):



![plot of chunk predicted infections through time]({{ site.url }}/assets/20200421-114817-ece7e7f5/cumu_inf_exp_model.png)

*Figure 7: Cumulative estimated infections. The dashed line indicates the date of last sample in Liège in this analysis. The points represent reported cases in Liège.*

Based on an estimated growth rate of 0.220 [0.187 - 0.247] median [95% CrI]:  

We estimate cumulative number of infections at last sample (2020-03-31) as: 113838 [25519 - 690359]

We estimate number of infections at 2020-04-10 (10 days after last sample) as:
1040182 [164414 - 7399452]  




## Methods summary



Details on methods and priors can be [found here](http://whoinfectedwhom.org/seijr0.1.0_methods.pdf).


Model version: seijr0.1.0

Report version: 20200421-114817-ece7e7f5


## Acknowledgements

This work was supported by the [MRC Centre for Global Infectious Disease Analysis at Imperial College London](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis).

Sequence data were provided by [GISAID](http://www.epicov.org) and [these laboratories](http://whoinfectedwhom.org/gisaid_cov2020_acknowledgement_table.xls).


