---
urlcolor: blue
linkcolor: blue
output:
  word_document:
    fig_width: 7
    fig_height: 5
    fig_caption: true
    highlight: "tango"
    reference_docx: word_style.docx
    df_print: "kable"
  html_document:
    fig_width: 7
    fig_height: 5
    fig_caption: true
    highlight: "tango"
---








---
title: "Phylodynamic Analysis: Auvergne-Rhône-Alpes, 2020-03-24 "
date: April 14, 2020
---





### Primary author: Olivia Boyd

### Report prepared on 2020-04-14

#### On behalf of the MRC GIDA COVID-19 phylodynamics working group at Imperial College London: Lily Geidelberg, Olivia Boyd, Manon Ragonnet, David Jorgensen,  Igor Siveroni, Erik Volz




## Background information  




#### This is analysis is based on : 
  
* **43 whole genomes** sampled from **within ARA**
* **66 whole genomes** sampled from outside of **ARA**
* Samples within ARA were collected between **2020-03-15** and **2020-03-24**

These numbers will differ from the number of uploaded sequences as we remove sequences with likely sequencing errors or significant gaps. Sequences were deduplicated prior to analyses. 

<!-- ##### To add: [optional plot of sample distribution through time] -->



## How many are infected in ARA?

Using a phylodynamic model we estimate epidemiological parameters using SARS-CoV 2 sequence data from Liège together with a background set of sequences sampled from the larger internationational viral population. [The model is explained in detail here](http://whoinfectedwhom.org/seijr0.1.0_methods.pdf). Reported case numbers for comparison are extracted from [Santé publique France](https://www.santepubliquefrance.fr/maladies-et-traumatismes/maladies-et-infections-respiratoires/infection-a-coronavirus/articles/infection-au-nouveau-coronavirus-sars-cov-2-covid-19-france-et-monde).




![plot of chunk Cumulative estimated infections through time]({{ site.url }}/assets/20200414-100752-6c17af8a/SEIJR_plot_size.png)


![plot of chunk Cumulative estimated infections through time log scale]({{ site.url }}/assets/20200414-100752-6c17af8a/SEIJR_plot_size_log.png)


*Figure 1: Estimated cumulative infections through time represented by solid black line (median) and 95% CrI (ribbon). Black points represent reported cases in ARA. The dashed line indicates the date of last sample in ARA in this analysis.*


* Estimated cumulative infections at last sample (2020-03-24): **146356 [1-1514725]** median [95%CI]

* Cumulative confirmed infections reported at 2020-03-24: **1991**  

<!-- * Cumulative number of active infections at 2020-03-24:   -->



![plot of chunk daily estimated infections through time]({{ site.url }}/assets/20200414-100752-6c17af8a/Daily.png)


![plot of chunk daily estimated infections through time log scale]({{ site.url }}/assets/20200414-100752-6c17af8a/Daily_log.png)


*Figure 2: Estimated daily  infections through time represented by solid black line (median) and 95% CrI (ribbon). Black points represent reported cases in ARA. The dashed line indicates the date of last sample in ARA in this analysis.*



![plot of estimated reported rate]({{ site.url }}/assets/20200411-191528-8ad25e14/reporting2.png)

*Figure 3: Estimated daily reporting rate of cases in Liège. Note that the CI of reporting rate (not shown) extends to 100% on most days due to very high imprecision of epidemic size estimates.*


The percentage of estimated cases which were confirmed reported cases between 2020-03-01 and 2020-03-31 ranged from 23.6% to 2.98% over time with a mean of 15.27%. However there is very high uncertainty in these estimates corresponding to imprecision of epidemic size estimates. The mean estimated reporting rate trend is similar to previous findings [reported by Russell et al using case fatality ratio estimates.](https://cmmid.github.io/topics/covid19/severity/global_cfr_estimates.html) 


![plot of chunk Rt]({{ site.url }}/assets/20200414-100752-6c17af8a/Rt.png)

*Figure 3: Reproduction number through time. The black vertical dashed line indicates the date of last sample in ARA in this analysis. Orange and red dashed lines indicate dates of school closure and general lockdown in ARA, respectively. *

Reproduction number at last sample (2020-03-24): **3.55 [0.464-4.3]** median [95% CrI]


## How quickly has the epidemic in ARA grown?




```


| Quantile | Reproduction number | Growth rate (per day) | Doubling time (days) |
|:--------:|:-------------------:|:---------------------:|:--------------------:|
|   50%    |        3.82         |         0.242         |         2.71         |
|   2.5%   |        0.475        |        -0.0766        |        -12.5         |
|  97.5%   |        5.08         |         0.319         |         3.53         |

Table: Reproduction number, growth rate and doubling times
```






## How has SARS-CoV 2 evolved in ARA?



![plot of chunk mcc_tree]({{ site.url }}/assets/20200414-100752-6c17af8a/mcc.png)

*Figure 4: Time scaled phylogeny co-estimated with epidemiological parameters. The colour of the tips corresponds to location sampling; red tips were sampled from within ARA, blue tips from outside.*



##### Molecular clock rate of evolution: **0.00112 [0.000816-0.0014]** median [95% CrI]  

<!-- #### (optional) Number of introductions into ARA (someone needs to write code to compute this) -->




## Predicted cumulative infections over next 14 days (assuming exponential growth):



![plot of chunk predicted infections through time]({{ site.url }}/assets/20200414-100752-6c17af8a/cumu_inf_exp_model.png)

*Figure 5: Cumulative estimated infections. The dashed line indicates the date of last sample in ARA in this analysis. The points represent reported cases in ARA.*

Based on an estimated growth rate of 0.242 [-0.0766 - 0.319] median [95% CrI]:  

We estimate cumulative number of infections at last sample (2020-03-24) as: 17315 [0 - 303349]

We estimate number of infections at 2020-04-08 (15 days after last sample) as:
656221 [0 - 36973092]  




## Methods summary



Details on methods and priors can be [found here](http://whoinfectedwhom.org/seijr0.1.0_methods.pdf).


Model version: seijr0.0.0

Report version: 20200414-100752-6c17af8a


## Acknowledgements

This work was supported by the [MRC Centre for Global Infectious Disease Analysis at Imperial College London](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis).

Sequence data were provided by [GISAID](http://www.epicov.org) and [these laboratories](http://whoinfectedwhom.org/gisaid_cov2020_acknowledgement_table.xls).


