---
urlcolor: blue
linkcolor: blue
output:
  word_document:
    fig_width: 7
    fig_height: 5
    fig_caption: true
    highlight: "tango"
    reference_docx: word_style.docx
    df_print: "kable"
  html_document:
    fig_width: 7
    fig_height: 5
    fig_caption: true
    highlight: "tango"
---








---
title: "Phylodynamic Analysis: Liège, 2020-03-31 "
date: April 11, 2020
---





### Primary author: Olivia Boyd

### Report prepared on 2020-04-11

#### On behalf of the MRC GIDA COVID-19 phylodynamics working group at Imperial College London: Lily Geidelberg, Olivia Boyd, Manon Ragonnet, David Jorgensen,  Igor Siveroni, Erik Volz




## Background information  




#### This analysis is based on : 
  
* **50 whole genomes** sampled from **within Liège**
* **73 whole genomes** sampled from outside of **Liège**
* Samples within Liège were collected between **2020-03-05** and **2020-03-31**


These numbers will differ from the number of uploaded sequences as we remove sequences with likely sequencing errors or significant gaps. Sequences were deduplicated prior to analyses. 


<!-- ##### To add: [optional plot of sample distribution through time] -->



## How many are infected in Liège?

Using a phylodynamic model we estimate epidemiological parameters using SARS-CoV 2 sequence data from Liège together with a background set of sequences sampled from the larger internationational viral population. [The model is explained in detail here](http://whoinfectedwhom.org/seijr0.1.0_methods.pdf). Reported case numbers for comparison are extracted from [Sciensano, the Belgian instute for health](https://epistat.wiv-isp.be/covid/).




![plot of chunk Cumulative estimated infections through time]({{ site.url }}/assets/20200411-191528-8ad25e14/SEIJR_plot_size.png)

*Figure 1: Cumulative estimated infections through time. Points represent reported cases in Liège. The dashed line indicates the date of last sample in Liège in this analysis.*


* Estimated cumulative infections at last sample (2020-03-31): **57257 [1-190723]** median [95%CI]

* Cumulative confirmed infections reported at 2020-03-31: **1854**

* Percent of estimated cumlutive infections (see Figure 3) which are reported confirmed infections at 2020-03-21: **3.24**

<!-- * Cumulative number of active infections at 2020-03-31:   -->



![plot of chunk daily estimated infections through time]({{ site.url }}/assets/20200411-191528-8ad25e14/Daily.png)

*Figure 2: Daily estimated infections through time. Points represent reported cases in Liège. The dashed line indicates the date of last sample in Liège in this analysis.*

Towards the end of the curve the estimated number of daily infections starts to decrease. This may be due to the shutting of all non-essential activities on 2020-03-18 across Belgium. However, further anlayses are required to assess the impact of non-pharmaceutical interventions on transmission in this region. 

![plot of estimated reported rate]({{ site.url }}/assets/20200411-191528-8ad25e14/reporting.png)

*Figure 3: Estimated daily reporting rate of cases in Liège. Error bars represent the 95% credible interval.*

The percentage of estimated daily reported cases which were confirmed reported cases between 2020-03-01 and 2020-03-31 ranged from 1.12 to 6.32% over time with a mean of 4.71%. The mean estimated reporting rate is similar to previous findings [reported by Russell et al using case fatality ratio estimates.](https://cmmid.github.io/topics/covid19/severity/global_cfr_estimates.html) 

![plot of chunk Rt]({{ site.url }}/assets/20200411-191528-8ad25e14/Rt.png)

*Figure 4: Reproduction number through time. The vertical dashed line indicates the date of last sample in Liège in this analysis.*

Reproduction number at last sample (2020-03-31): **0.912 [0.105-2.58]** median [95% CrI]


## How quickly has the epidemic in Liège grown?




```


| Quantile | Reproduction number | Growth rate (per day) | Doubling time (days) |
|:--------:|:-------------------:|:---------------------:|:--------------------:|
|   50%    |        3.43         |         0.216         |         3.1          |
|   2.5%   |        0.476        |        -0.0765        |        -11.2         |
|  97.5%   |        5.42         |         0.338         |         3.96         |

Table: Reproduction number, growth rate and doubling times
```






## How has SARS-CoV 2 evolved in Liège?



![plot of chunk mcc_tree]({{ site.url }}/assets/20200411-191528-8ad25e14/mcc.png)

*Figure 5: Time scaled phylogeny co-estimated with epidemiological parameters. The colour of the tips corresponds to location sampling; red tips were sampled from within Liège, blue tips from outside.*



##### Molecular clock rate of evolution: **0.000965 [0.000661-0.00117]** median [95% CrI]  

<!-- #### (optional) Number of introductions into Liège (someone needs to write code to compute this) -->




## Predicted cumulative infections over next 14 days (assuming exponential growth):



![plot of chunk predicted infections through time]({{ site.url }}/assets/20200411-191528-8ad25e14/cumu_inf_exp_model.png)

*Figure 6: Cumulative estimated infections. The dashed line indicates the date of last sample in Liège in this analysis. The points represent reported cases in Liège.*

Based on an estimated growth rate of 0.216 [-0.0765 - 0.338] median [95% CrI]:  

We estimate cumulative number of infections at last sample (2020-03-31) as: 87789 [0 - 19253250]

We estimate number of infections at 2020-04-10 (10 days after last sample) as: 743615 [0 - 516106723]  




## Methods summary



Details on methods and priors can be [found here](http://whoinfectedwhom.org/seijr0.1.0_methods.pdf).


Model version: seijr0.1.0

Report version: 20200411-191528-8ad25e14


## Acknowledgements

This work was supported by the [MRC Centre for Global Infectious Disease Analysis at Imperial College London](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis).

Sequence data were provided by [GISAID](http://www.epicov.org) and [these laboratories](http://whoinfectedwhom.org/gisaid_cov2020_acknowledgement_table.xls).


